### PR TITLE
Disable stack traces for exceptions from ratelimiter and bulkhead

### PR DESCRIPTION
**Pull request by worldtiki**
Related with #618.

Similar approach as the one used for the circuit breaker.

I'm not sure about what documentation/examples I need to update.

### CODE REVIEW

File: resilience4j-bulkhead/src/main/java/io/github/resilience4j/bulkhead/BulkheadFullException.java [hunk 1]
```diff
@@ -24,22 +24,32 @@
...
-     * The constructor with a message.
+     * Static method to construct a {@link BulkheadFullException} with a ThreadPoolBulkhead.
      *
      * @param bulkhead the Bulkhead.
      */
-    public BulkheadFullException(ThreadPoolBulkhead bulkhead) {
-        super(String.format("Bulkhead '%s' is full and does not permit further calls", bulkhead.getName()));
-    }
-}
+    public static BulkheadFullException getCallNotPermittedException(ThreadPoolBulkhead bulkhead) {
```
Comment (RobWin) [inline resilience4j-bulkhead/src/main/java/io/github/resilience4j/bulkhead/BulkheadFullException.java]: getBulkheadFullException

File: resilience4j-ratelimiter/src/main/java/io/github/resilience4j/ratelimiter/RequestNotPermitted.java [hunk 2]
```diff
@@ -32,4 +32,21 @@
...
     }
-}
\ No newline at end of file
+
+    /**
+     * Static method to construct a {@link RequestNotPermitted} with a ThreadPoolBulkhead.
+     *
+     * @param rateLimiter the RateLimiter.
+     */
+    public static RequestNotPermitted getCallNotPermittedException(RateLimiter rateLimiter) {
```
Comment (RobWin) [inline resilience4j-ratelimiter/src/main/java/io/github/resilience4j/ratelimiter/RequestNotPermitted.java]: getRequestNotPermitted

File: resilience4j-bulkhead/src/main/java/io/github/resilience4j/bulkhead/BulkheadFullException.java (hunk 1 above)
Comment (worldtiki) [reply resilience4j-bulkhead/src/main/java/io/github/resilience4j/bulkhead/BulkheadFullException.java]: Ooops :(

**Comment by RobWin** [comment]
Thanks

**Pull request merged**